eng/AfterSolutionBuild.targets

```diff
@@ -5,7 +5,8 @@
   <Import Project="SharedFramework.Local.props" />
 
   <!-- This is temporary until we can use FrameworkReference to build our own packages. -->
-  <Target Name="RemoveSharedFrameworkOnlyRefsFromNuspec" AfterTargets="Pack">
+  <Target Name="RemoveSharedFrameworkOnlyRefsFromNuspec" AfterTargets="Pack"
+      Condition=" '$(MSBuildRuntimeType)' == 'core' ">
```

Comment:
why was the condition added?

Reply:
A single `build.cmd` invocation now runs the `MSBuild` PowerShell function twice (three times including the repo tasks) and both would redo this step if the dev had built the repo earlier. Since the fixup only matters for managed packages and we only build managed packages in the `dotnet msbuild` invocation, this is safe and quiets things down.